env/production/fluentbit.yaml

```diff
@@ -331,10 +331,7 @@ spec:
         version: v1
         kubernetes.io/cluster-service: "true"
     spec:
-      initContainers:
-      - name: wait-for-init
-        image: busybox:1.28
-        command: ['sh', '-c', 'echo "Waiting for 2 seconds for node to sort itself out" && sleep 2']      
```

Comment:
oh wait we get rid of the 2 seconds waiting here? so we can see the tls error and investigate?

Reply:
and do we want this in prod too or just staging?

Reply:
The interwebs and SO say that this is pretty much the solution to getting rid of this error... I want to see if it does in our case. 

As for removing the wait in production, 2 seconds wasn't long enough so I got rid of it... I can add it back or even up it to 10 seconds here and see if that helps for prod

Reply:
Let's test this in staging and see how the experiments goes. We can revert back to what worked best then. 👍